gulpfile.js

```diff
@@ -1,6 +1,7 @@
 var gulp = require('gulp');
 var concat = require('gulp-concat');
 var sass = require('gulp-sass');
+var less = require('gulp-less');
```

Comment:
Yeah, that one also has the Glyph Icon font included. With some minor changes it would be changed I guess.